sys/shell/commands/sc_netif.c

```diff
@@ -989,11 +1006,36 @@ int _netif_config(int argc, char **argv)
             }
 #ifdef MODULE_NETSTATS_L2
```

Comment:
+ `MODULE_NETSTATS_IPV6` (or `MODULE_NETSTATS_PRINT` if you wanna follow my advice from above)